taipy/core/job/job.py

```diff
@@ -136,6 +136,11 @@ def creation_date(self, val):
     def stacktrace(self) -> List[str]:
```

Comment:
I'm not sure of the reason why but if you want to make `stacktrace` auto set, you might want to add auto reload to its attribute also by adding `@_self_reload(_MANAGER_NAME)` after `@property` here

Reply:
Done!